src/pages/spawn/spawnHost/SpawnHostCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { Link } from "react-router-dom";
+import { SiderCard } from "components/styles";
+import { routes } from "constants/routes";
+import { Host } from "gql/generated/types";
+import { getDateCopy } from "utils/string";
+
+interface SpawnHostCardProps {
+  host: Host;
+}
+
+export const SpawnHostCard: React.FC<SpawnHostCardProps> = ({ host }) => (
+  <StyledSiderCard data-cy="spawn-host-card">
+    {Object.keys(spawnHostCardFieldMaps).map((key) => (
+      <SpawnHostEntry
+        key={`${key}_${spawnHostCardFieldMaps[key](host)}`}
```

Comment:
`${spawnHostCardFieldMaps[key](host)}` will evaluate to `[object Object]`

Reply:
Nice catch!